espflash/src/partition_table.rs

```diff
@@ -209,29 +336,62 @@ impl<W: Write> HashWriter<W> {
     }
 }
 
-#[test]
-fn test_basic() {
-    use std::fs::read;
-    const NVS_ADDR: u32 = 0x9000;
-    const PHY_INIT_DATA_ADDR: u32 = 0xf000;
-    const APP_ADDR: u32 = 0x10000;
-
-    const NVS_SIZE: u32 = 0x6000;
-    const PHY_INIT_DATA_SIZE: u32 = 0x1000;
-    const APP_SIZE: u32 = 0x3f0000;
-
-    let expected = read("./tests/data/partitions.bin").unwrap();
-    let table = PartitionTable::basic(
-        NVS_ADDR,
-        NVS_SIZE,
-        PHY_INIT_DATA_ADDR,
-        PHY_INIT_DATA_SIZE,
-        APP_ADDR,
-        APP_SIZE,
-    );
-
-    let result = table.to_bytes();
-
-    assert_eq!(expected.len(), result.len());
-    assert_eq!(expected, result.as_slice());
+#[allow(unused)]
```

Comment:
For some reason without this line  the `use super::*;` line as well as the `PTABLE_0` and `PTABLE_1` constants get marked as unused, and fail the clippy checks. Has anybody experienced this before?

Reply:
the `unused` check is done without the `test` cfg, so anything that is only used by tests is seen as unused, adding a `#[cfg(test)]` should fix this

Reply:
Oops, forgot about that. Thanks!